### DISCUSSION TITLE
Using ansible-navigator with password prompts

### DISCUSSION THREAD

**Discussion by Seandersont** [ansible/ansible-navigator#589] [Q&A]
When I try running a playbook with a prompt such as for a vault password or a administrator password, it's not clear on how to get into the prompt to pass that information to the playbook, unless I put it into a password file/config file. Is this currently a feature built into navigator, and if so, how would I best go about this?

**Comment by cidrblock** [comment]
Thanks for asking! Check out the question in the FAQ about vars_prompt and LMK if that explains how to get it to work. https://ansible-navigator.readthedocs.io/en/latest/faq/

I should mention we don't have any plans to support a run time prompt in the TUI b/c it's not something awx/controller do....

TY again. Hope you are enjoying navigator!
